benchcab/benchcab.py

```diff
@@ -290,6 +290,9 @@ def main(self):
         if self.args.subcommand == "run":
             self.run()
 
+        if self.args.subcommand == "validate_config":
+            pass  # handle validation in __init__ method
```

Comment:
As per yesterdays meeting, there is a validate_config method now present in config.py.

https://github.com/CABLE-LSM/benchcab/blob/42e66b693a4fdfcb3f314f447eb31081a671c003/benchcab/config.py#L32

Returns True if valid and raises an exception if not.

It would be worthwhile wrapping this in a local method to return something useful for the user in either case.